styles/style.css

```diff
@@ -0,0 +1,56 @@
+* {
+  box-sizing: border-box;
+  font-family: "Courier New", Courier, monospace;
+  /* border: 1px solid red; */
+}
+
+h1 {
+  text-align: center;
+  padding-bottom: 25px;
+  background-color: aliceblue;
+  margin-left: 10px;
```

Comment:
brauchst du gleichen Margin left und right, wenn du text-align: center hast? (auch in der nächsten Klasse)

Reply:
jup, stimmt !